crates/rspack_core/src/build_chunk_graph/code_splitter.rs

```diff
@@ -217,6 +217,11 @@ impl<'me> CodeSplitter<'me> {
       self.remove_parent_modules();
     }
 
+    // make sure all module (weak dependency particularly) has a mgm
+    for module_identifier in self.compilation.module_graph.modules().keys() {
+      self.compilation.chunk_graph.add_module(*module_identifier)
+    }
```

Comment:
Weak dependency will [not be traveled in buildChunkGraph](https://github.com/webpack/webpack/blob/9f0480537ff7460ae854d91783b2c2f4b8e51ac6/lib/buildChunkGraph.js#L100), its mgm will be added at the first time `chunkGraph._getChunkGraphModule` called, but in Rspack, mgm will be added by `chunkGraph.add_module`, so I add these lines to make sure mgm is existed, feel free to refactor these if you think there is better place for these lines @hyf0